linker/Mono.Linker/Driver.cs

```diff
@@ -77,109 +77,110 @@ bool HaveMoreTokens ()
 		void Run ()
 		{
 			Pipeline p = GetStandardPipeline ();
-			LinkContext context = GetDefaultContext (p);
-			I18nAssemblies assemblies = I18nAssemblies.All;
-			var custom_steps = new List<string> ();
-
-			bool resolver = false;
-			while (HaveMoreTokens ()) {
-				string token = GetParam ();
-				if (token.Length < 2)
-					Usage ("Option is too short");
-
-				if (! (token [0] == '-' || token [1] == '/'))
-					Usage ("Expecting an option, got instead: " + token);
-
-				if (token [0] == '-' && token [1] == '-') {
-
-					if (token.Length < 3)
+			using (LinkContext context = GetDefaultContext (p)) {
```

Comment:
All I did was add this using and then indent everything.   I double checked the indents and end of lines, they are all good.  I'm not sure why the diff is so ugly.

The code style is fine.  There was one change on line 90 with the space after the !, but I'd argue that was wrong to begin with since the more common pattern used in the code base seems to be not using a space.